forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/ProtectCallbackTest.kt

```diff
@@ -42,8 +42,8 @@ class ProtectCallbackTest {
         ) {
             val nodeListener: NodeListener<FRSession?> = this
             override fun onCallbackReceived(node: Node) {
-                if (node.getCallback<PingOneProtectInitCallback>(PingOneProtectInitCallback::class.java) != null) {
-                    val callback: PingOneProtectInitCallback = node.getCallback<PingOneProtectInitCallback>(PingOneProtectInitCallback::class.java)
+                if (node.getCallback<PingOneProtectInitializeCallback>(PingOneProtectInitializeCallback::class.java) != null) {
```

Comment:
(node.getCallback<PingOneProtectInitializeCallback>(PingOneProtectInitializeCallback::class.java)?.let {

}

just one update here as well ...all good

Reply:
Oh, this file actually has to be removed...